pypdf/_cmap.py

```diff
@@ -258,7 +257,8 @@ def prepare_cm(ft: DictionaryObject) -> bytes:
     tu = ft["/ToUnicode"]
     cm: bytes
     if isinstance(tu, StreamObject):
-        cm = b_(cast(DecodedStreamObject, ft["/ToUnicode"]).get_data())
+        cm = cast(DecodedStreamObject, ft["/ToUnicode"]).get_data()
+        assert isinstance(cm, bytes)
```

Comment:
I recommend throwing a specific *pypdf* exception instead of an `AssertionError`.

Reply:
this was only for dev/debug. To be removed